vagrant/plugins.rb

```diff
@@ -1,6 +1,6 @@
 # Install plugins
 
-required_plugins = %w( vagrant-hosts-provisioner vagrant-notify-forwarder )
+required_plugins = %w( vagrant-hosts-provisioner )
```

Comment:
Don't we need vagrant-notify-forwarder any more?

Reply:
we are not using it for anything after all.